staging/src/k8s.io/kube-aggregator/pkg/registry/apiservice/rest/storage_apiservice.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 // NewRESTStorage returns an APIGroupInfo object that will work against apiservice.
-func NewRESTStorage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter, shouldServeBeta bool) genericapiserver.APIGroupInfo {
+func NewRESTStorage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) genericapiserver.APIGroupInfo {
```

Comment:
Oh man, so this is a staging repo and this is an exported function, so backward compatibility rules should apply.  But the param is not used in the function.

Unless we're super sure we know how everyone uses this, I favor we document that the param is not used but leave the signature unchanged.

Reply:
cc @deads2k, to maintain backwards compatibility we're keeping the parameter, and fixing the usage in this PR.